From 864d2577fb2558c12cb9499567b3bb76d7058ec8 Mon Sep 17 00:00:00 2001
From: Eric Biggers <ebiggers@google.com>
Date: Wed, 12 Aug 2020 13:59:59 -0700
Subject: [PATCH 100/830] _____ANNOTATION: iommu changes_____
X-NVConfidentiality: public

Signed-off-by: Eric Biggers <ebiggers@google.com>
---
 ANNOTATIONS | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ANNOTATIONS b/ANNOTATIONS
index 63468c0f9120..96ed870f7c7e 100644
--- a/ANNOTATIONS
+++ b/ANNOTATIONS
@@ -3,3 +3,4 @@ Android vendor hooks
 kbuild changes
 x86 changes
 arm and arm64 changes
+iommu changes
-- 
2.17.1

