From d8ebe10413d05c737e63a386d778649961c65e85 Mon Sep 17 00:00:00 2001
From: Eric Biggers <ebiggers@google.com>
Date: Wed, 12 Aug 2020 14:00:25 -0700
Subject: [PATCH 219/830] _____ANNOTATION: mm (memory management) changes_____
X-NVConfidentiality: public

Signed-off-by: Eric Biggers <ebiggers@google.com>
---
 ANNOTATIONS | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ANNOTATIONS b/ANNOTATIONS
index 79a13deeaef4..45434eb17fda 100644
--- a/ANNOTATIONS
+++ b/ANNOTATIONS
@@ -9,3 +9,4 @@ power management changes
 scheduler changes
 uid_sys_stats support
 cpufreq changes
+mm (memory management) changes
-- 
2.17.1

