From 598eedfa8ded44f83c3fcc3591f1dafa1ab8a367 Mon Sep 17 00:00:00 2001
From: Eric Biggers <ebiggers@google.com>
Date: Wed, 12 Aug 2020 13:57:54 -0700
Subject: [PATCH 233/830] _____ANNOTATION: binder changes_____
X-NVConfidentiality: public

Signed-off-by: Eric Biggers <ebiggers@google.com>
---
 ANNOTATIONS | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ANNOTATIONS b/ANNOTATIONS
index 03eb4490d7bd..1c003b0be641 100644
--- a/ANNOTATIONS
+++ b/ANNOTATIONS
@@ -11,3 +11,4 @@ uid_sys_stats support
 cpufreq changes
 mm (memory management) changes
 security changes
+binder changes
-- 
2.17.1

