From e803c5ea8b146f839b7d398066284381d6dc76bc Mon Sep 17 00:00:00 2001
From: Eric Biggers <ebiggers@google.com>
Date: Wed, 12 Aug 2020 14:00:03 -0700
Subject: [PATCH 238/830] _____ANNOTATION: ion changes_____
X-NVConfidentiality: public

Signed-off-by: Eric Biggers <ebiggers@google.com>
---
 ANNOTATIONS | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ANNOTATIONS b/ANNOTATIONS
index 1c003b0be641..b3372ce3e435 100644
--- a/ANNOTATIONS
+++ b/ANNOTATIONS
@@ -12,3 +12,4 @@ cpufreq changes
 mm (memory management) changes
 security changes
 binder changes
+ion changes
-- 
2.17.1

