From 5c08bc84ad09c064d9312dc9f5b6647c54d1f4cb Mon Sep 17 00:00:00 2001
From: "Steven Rostedt (VMware)" <rostedt@goodmis.org>
Date: Wed, 25 Aug 2021 21:16:57 -0400
Subject: [PATCH 304/304] Linux 5.10.65-rt53 REBASE

---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 21988f9..e6c0dc6 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt34
+-rt53
-- 
2.7.4

